meta/execution-environment.yml

```diff
@@ -16,6 +16,7 @@
 # not needed for end-users to use the collection. Ansible-builder installing these
 # would increase the EE size and increase the chance of dependency collisions.
 
+version: 3
```

Comment:
This is wrong. This isn't an execution environment **definition**, but execution environment **collection-level metadata**: https://ansible.readthedocs.io/projects/builder/en/latest/collection_metadata/
It does not have a `version` field.
```suggestion
```

Reply:
I see! TIL - Interesting, Ansible Lint is picking this file up as being an Execution Environment definition: https://ansible.readthedocs.io/projects/builder/en/latest/definition/

Will look to raise this on Ansible Lint later today.

Reply:
I raised https://github.com/ansible/ansible-lint/issues/4710 for this. I've correct this now. Thanks @felixfontein